### PR TITLE
Add partition info status JSON, and wrap it in HTML for browsers

### PR DESCRIPTION
**Pull request by colinmarc**
This adds partition info to the status JSON returned by `GET /` and
`GET /dbname/`. It also adds a nice html version, for browsers.

### CODE REVIEW

**Comment by colinmarc** [comment]
The status page looks like this: 

Feels like it's missing something...

File: status.tmpl [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<html>
+  <head>
+    <style type="text/css">
```
Comment (forklady42) [inline status.tmpl]: Let's break the css into a separate file, in order to clean this it up a bit. (Makes it easier to focus on just the template.)

Comment (colinmarc) [inline status.tmpl]: It's less than 100 lines =) If we do that, we have to chunk the CSS file into the binary too, and then write an HTTP handler for it...

**Comment by colinmarc** [comment]
Ok, much nicer, I think:

**Comment by forklady42** [comment]
looks good! I like the new layout. this will be sooooo helpful 👍 

(boo flakey test, but it's just flakey)

**Pull request merged**